server/app/v1/pokemons/mappers/pokemons.mappers.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  DetailApiResponse,
+  PokemonDetail,
+  PokemonApiResponse,
+  TransformedPokemonList,
+} from '~/type'
+
+export const transformPokemonList = (
+  res: PokemonApiResponse,
+  { page, pageSize }: { page: number; pageSize: number }
+): TransformedPokemonList => {
+  const pokemons = res.results.map(pokemon => {
+    const id = parseInt(pokemon.url.split('/').slice(-2, -1)[0])
+    return {
+      id,
+      name: pokemon.name,
+      image: `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${id}.png`,
+    }
+  })
+  return {
+    count: res.count,
+    page,
+    pageSize,
+    totalPages: Math.ceil(res.count / pageSize),
+    pokemons,
+  }
+}
+
+export const transformPokemonDetail = (
+  pokemon: DetailApiResponse
+): PokemonDetail => {
+  return {
+    id: pokemon.id,
+    name: pokemon.name,
+    image: pokemon.sprites.front_default,
```

Comment:
the name `image` is bad. Why? cause it is very unclear what type. 

Is it the image as a file? is it image as base64 string? is it the image URL?